content/source/docs/enterprise/workspaces/access.html.md

```diff
@@ -8,7 +8,7 @@ sidebar_current: "docs-enterprise2-workspaces-access"
 
 Terraform Enterprise (TFE) workspaces can only be accessed by users with the correct permissions. You can manage permissions for a workspace on a per-team basis.
 
-Managing access requires [admin privileges](../users-teams-organizations/permissions.html) on the affected workspace.
+Workspace access should be managed by [organization owners](../users-teams-organizations/teams.html#the-owners-team). (Although any user with [admin privileges](../users-teams-organizations/permissions.html) on the target workspace can edit its permissions, non-owners can only add permissions for teams they are members of.)
```

Comment:
I think "on a workspace" makes more sense here than "on the target workspace".

Reply:
What's the reasoning for this tweak? Honestly I think the original is more clear to me than the change (I had to reread a bunch to figure out the distinction you're making here, but then it seems obvious now that I understand it). It seems like this is just a recommendation. As an owner user, isn't it plausible that I give admin access to someone with the intention of them managing that workspace's access for others thereafter?

Reply:
I went with 

> Workspace access should be managed by [organization owners](../users-teams-organizations/teams.html#the-owners-team). (Users with [admin privileges](../users-teams-organizations/permissions.html) on a workspace can make minor adjustments to its permissions, but only owners can manage permissions for any team in the organization.)